# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/mediapipeline/MediaPipelineFilter.cpp
# Commit: 6a52b92ff037
# Full Hash: 6a52b92ff03731a1e2ef13aa7f7349a1e7e48bee
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 1
# Description:
#   Bug 1376873 - Mediapipeline updates; r=pehrsons
#   
#   Support for native_handle() has been removed by upstream.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D7437
# ==============================================================================

diff -r 6a6771656fe5 -r 6a52b92ff037 media/webrtc/signaling/src/mediapipeline/MediaPipelineFilter.cpp
--- a/media/webrtc/signaling/src/mediapipeline/MediaPipelineFilter.cpp	Thu Feb 15 15:14:58 2018 -0500
+++ b/media/webrtc/signaling/src/mediapipeline/MediaPipelineFilter.cpp	Fri Feb 16 09:15:44 2018 -0500
@@ -36,16 +36,16 @@
     return false;
   }
 
-  if (!header.extension.rtpStreamId.empty() &&
+  if (!header.extension.stream_id.empty() &&
       !remote_rid_set_.empty() &&
-      remote_rid_set_.count(header.extension.rtpStreamId.data())) {
+      remote_rid_set_.count(header.extension.stream_id.data())) {
     return true;
   }
-  if (!header.extension.rtpStreamId.empty()) {
+  if (!header.extension.stream_id.empty()) {
     MOZ_LOG(gMediaPipelineLog, LogLevel::Debug,
             ("MediaPipelineFilter ignoring seq# %u ssrc: %u RID: %s",
              header.sequenceNumber, header.ssrc,
-             header.extension.rtpStreamId.data()));
+             header.extension.stream_id.data()));
   }
 
   if (remote_ssrc_set_.count(header.ssrc)) {
